mmcls/models/backbones/efficientnet_v2.py

```diff
@@ -0,0 +1,395 @@
+
+            self.project_conv = ConvBNAct(
+                expanded_c,
+                out_c,
+                kernel_size=1,
+                norm_layer=norm_layer,
+                activation_layer=nn.Identity)  # 注意没有激活函数
+        else:
+            # 当只有project_conv时的情况
+            self.project_conv = ConvBNAct(
+                input_c,
+                out_c,
+                kernel_size=kernel_size,
+                stride=stride,
+                norm_layer=norm_layer,
+                activation_layer=activation_layer)  # 注意有激活函数
+
+        self.out_channels = out_c
+
+        # 只有在使用shortcut连接时才使用dropout层
+        self.drop_rate = drop_rate
+        if self.has_shortcut and drop_rate > 0:
+            self.dropout = DropPath(drop_rate)
+        else:
+            self.dropout = nn.Identity()
+
+    def forward(self, x: Tensor) -> Tensor:
+        if self.has_expansion:
+            result = self.expand_conv(x)
+            result = self.project_conv(result)
+        else:
+            result = self.project_conv(x)
+
+        if self.has_shortcut:
+            if self.drop_rate > 0:
+                result = self.dropout(result)
+
+            result += x
+
+        return result
+
+
+@MODELS.register_module()
+class EfficientNetV2(BaseBackbone):
+    # repeat, kernel, stride, expansion, in_c, out_c, se_ratio, operator
+    arch_settings = {
+        'base': [[1, 3, 1, 1, 16, 16, 0, 0], [2, 3, 2, 4, 16, 32, 0, 0],
+                 [2, 3, 2, 4, 32, 48, 0, 0], [3, 3, 2, 4, 48, 96, 0.25, 1],
+                 [5, 3, 1, 6, 96, 112, 0.25, 1],
+                 [8, 3, 2, 6, 112, 192, 0.25, 1]],
+        's': [[2, 3, 1, 1, 24, 24, 0, 0], [4, 3, 2, 4, 24, 48, 0, 0],
+              [4, 3, 2, 4, 48, 64, 0, 0], [6, 3, 2, 4, 64, 128, 0.25, 1],
+              [9, 3, 1, 6, 128, 160, 0.25, 1],
+              [15, 3, 2, 6, 160, 256, 0.25, 1]],
+        'm': [[3, 3, 1, 1, 24, 24, 0, 0], [5, 3, 2, 4, 24, 48, 0, 0],
+              [5, 3, 2, 4, 48, 80, 0, 0], [7, 3, 2, 4, 80, 160, 0.25, 1],
+              [14, 3, 1, 6, 160, 176, 0.25,
+               1], [18, 3, 2, 6, 176, 304, 0.25, 1],
+              [5, 3, 1, 6, 304, 512, 0.25, 1]],
+        'l': [[4, 3, 1, 1, 32, 32, 0, 0], [7, 3, 2, 4, 32, 64, 0, 0],
+              [7, 3, 2, 4, 64, 96, 0, 0], [10, 3, 2, 4, 96, 192, 0.25, 1],
+              [19, 3, 1, 6, 192, 224, 0.25,
+               1], [25, 3, 2, 6, 224, 384, 0.25, 1],
+              [7, 3, 1, 6, 384, 640, 0.25, 1]],
+        'xl': [[4, 3, 1, 1, 32, 32, 0, 0], [8, 3, 2, 4, 32, 64, 0, 0],
+               [8, 3, 2, 4, 64, 96, 0, 0], [16, 3, 2, 4, 96, 192, 0.25, 1],
+               [24, 3, 1, 6, 192, 256, 0.25, 1],
+               [32, 3, 2, 6, 256, 512, 0.25, 1],
+               [8, 3, 1, 6, 512, 640, 0.25, 1]],
+    }
+
+    def __init__(self,
+                 model_cnf: str = 's',
+                 num_features: int = 1280,
+                 drop_connect_rate: float = 0.0,
+                 frozen_stages: int = 0,
```

Comment:
please support `out_out_indices`  as https://github.com/open-mmlab/mmclassification/blob/2495400a98a3da9bd8cfb3731a711f4c77cc56a3/mmcls/models/backbones/efficientnet.py#L164

Reply:
for arch s/b0/b1/b2/b3 , the number of layers is 8; but for arch m/l/xl , the number of layers is 9, how to set the out_indices?